hangman/hangman.c

```diff
@@ -1,20 +1,182 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <ctype.h>
 
-char HANG_STATES[7][10 * 9] = 
+char HANG_STAGES[7][10 * 9] = 
 {
-	"             +         +----     +----     +----     +----     +----     +----     +----  ",
-	"             |         |         |   O     |   O     |   O     |   O     |   O     |   O  ",
-	"             |         |         |         |   +     | --+     | --+--   | --+--   | --+--",
-	"             |         |         |         |   |     |   |     |   |     |   |     |   |  ",
-	"             |         |         |         |         |         |         |  /      |  / \\ ",
-	"             |         |         |         |         |         |         |         |      ",
-	"/*****\\   /*****\\   /*****\\   /*****\\   /*****\\   /*****\\   /*****\\   /*****\\   /*****\\   "
+	"         +        +----    +----    +----    +----    +----    +----    +----  ",
+	"         |        |        |   O    |   O    |   O    |   O    |   O    |   O  ",
+	"         |        |        |        |   +    | --+    | --+--  | --+--  | --+--",
+	"         |        |        |        |   |    |   |    |   |    |   |    |   |  ",
+	"         |        |        |        |        |        |        |  /     |  / \\ ",
+	"         |        |        |        |        |        |        |        |       ",
+	"/*****\\  /*****\\  /*****\\  /*****\\  /*****\\  /*****\\  /*****\\  /*****\\  /*****\\"
 };
 
+// Display Hang States
+void display_stage(int stage) {
+	int start = 9 * stage;
+	int end = start + 9;
+	printf("\n");
```

Comment:
great code Brock, I can see why you were having an easy time yesterday! :) awesome job!